.github/workflows/functional_test_v2.yaml

```diff
@@ -488,3 +488,270 @@ jobs:
+        with:
+          node_image: kindest/node:v1.30.0
+          kubectl_version: v1.30.0
+          cluster_name: kind
+          config: ./.github/workflows/configs/kind-config.yaml
+      - name: Fix kubelet TLS server certificates
+        run: |
+          kubectl get csr -o=jsonpath='{range.items[?(@.spec.signerName=="kubernetes.io/kubelet-serving")]}{.metadata.name}{" "}{end}' | xargs kubectl certificate approve
+      - name: Update dependencies
+        run: |
+          cd base && make dep-update
+      - name: Deploy cert-manager
+        run: |
+          cd base && make cert-manager
+      - name: install helm chart with operator enabled
+        run: |
+          cd base && helm install sock helm-charts/splunk-otel-collector --debug \
+          --set splunkObservability.realm=us0 --set splunkObservability.accessToken=xxxxx \
+          --set operator.enabled=true --set environment=dev --set clusterName=dev
+        continue-on-error: true
+      - name: check crds are installed
+        run: |
+          kubectl get crds | grep 'opentelemetry.io'
+
+          echo "get instrumentations - none if install failed"
+          kubectl get instrumentations.opentelemetry.io -A
+      - name: check operator
+        run: |
+          kubectl describe po -lapp.kubernetes.io/name=operator
+          kubectl get po -lapp.kubernetes.io/name=operator -o=name | xargs -n 1 kubectl wait --for=condition=Ready --timeout=2m
+          kubectl describe po -lapp.kubernetes.io/name=operator
+        continue-on-error: true
+      - name: upgrade old again
+        run: |
+          cd base && helm upgrade sock helm-charts/splunk-otel-collector --debug \
+          --set splunkObservability.realm=us0 --set splunkObservability.accessToken=xxxxx \
+          --set operator.enabled=true --set environment=dev --set clusterName=dev
+        continue-on-error: true
+      - name: check helm release status
+        run: |
+          helm status sock
+      - name: check instrumentations installed
+        run: |
+          kubectl describe instrumentations.opentelemetry.io -A
+      - name: Update dependencies
+        run: |
+          make dep-update
+      - name: try to import instrumentation in helm
+        run: |
+          NAMESPACE=default
+          RELEASE_NAME=sock
+          kubectl get -n $NAMESPACE instrumentations.opentelemetry.io -o name | xargs -I % kubectl annotate -n $NAMESPACE % meta.helm.sh/release-name=$RELEASE_NAME
+          kubectl get -n $NAMESPACE instrumentations.opentelemetry.io -o name | xargs -I % kubectl annotate -n $NAMESPACE % meta.helm.sh/release-namespace=$NAMESPACE
+      - name: upgrade helm chart
+        run: |
+          helm upgrade sock helm-charts/splunk-otel-collector --debug \
+          --set splunkObservability.realm=us0 --set splunkObservability.accessToken=xxxxx \
+          --set operator.enabled=true --set operator.crds.create=true --set environment=dev --set clusterName=dev
```

Comment:
`--set operator.crds.create=true` is the extra value to be set

Reply:
Could we also recommend to use `--reuse-values` here? https://helm.sh/docs/helm/helm_upgrade/

Reply:
no, I tried it in one of my earlier runs. The operator schema seems to have an update between the 2 versions which makes it fail validation with the option. I think it will also be a problem with the new values vars we have introduced since last version